pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -19,14 +19,14 @@ package v1alpha1
 import (
 	"context"
 	"fmt"
-
 	kapi "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	crapi "k8s.io/cluster-registry/pkg/apis/clusterregistry/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8client "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Yeah, a mistake. weird how that double import did not raise an error.  I aliased it because I wanted _client_ for a parameter name.  I'll change to k8sclient per convention you suggested.

Reply:
Fixed